openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/oracle/index.md

```diff
@@ -40,18 +40,44 @@ CREATE ROLE new_role;
 -- GRANT ROLE TO USER 
 GRANT new_role TO user_name;
 
--- GRANT CREATE SESSION PRIVILEGE TO USER
+-- Grant CREATE SESSION Privilege.
+--   This allows the role to connect.
 GRANT CREATE SESSION TO new_role;
 
--- GRANT SELECT CATALOG ROLE PRIVILEGE TO FETCH METADATA TO ROLE / USER
+-- Grant SELECT_CATALOG_ROLE Privilege.
+--   This allows the role ReadOnly Access to Data Dictionaries
 GRANT SELECT_CATALOG_ROLE TO new_role;
 ```
 
-With just these permissions, your user should be able to ingest the schemas, but not the tables inside them. To get
-the tables, you should grant `SELECT` permissions to the tables you are interested in. E.g.,
+If you don't want to create a role, and directly give permissions to the user, you can take a look at an example given below.
 
 ```sql
-SELECT ON ADMIN.EXAMPLE_TABLE TO new_role;
+-- Create a New User
+CREATE USER my_user IDENTIFIED by my_password;
+
+-- Grant CREATE SESSION Privilege.
+--   This allows the user to connect.
+GRANT CREATE SESSION TO my_user;
+
+-- Grant SELECT_CATALOG_ROLE Privilege.
+--   This allows the user ReadOnly Access to Data Dictionaries
+GRANT SELECT_CATALOG_ROLE to my_user;
+```
+
+**Note**: With just these permissions, your user should be able to ingest the metadata, but not the `Profiler & Data Quality`, you should grant `SELECT` permissions to the tables you are interested in for the `Profiler & Data Quality` features to work. 
+
+```sql
+-- if you are using role
+GRANT SELECT ON ADMIN.EXAMPLE_TABLE TO new_role;
```

Comment:
should we pass the `GRANT SELECT ON ANY` wording as well as a hint?

Reply:
I will add one more for a hint, we can keep an example for the specific table as well.